acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/permissioning/NodesWhitelistAcceptanceTest.java

```diff
@@ -59,12 +60,25 @@ public void permissionedNodeShouldDiscoverOnlyAllowedNodes() {
   public void permissionedNodeShouldDisconnectFromNodeRemovedFromWhitelist() {
     permissionedNode.verify(net.awaitPeerCount(2));
 
-    // remove allowed node from the whitelist
+    // remove node from the whitelist
     permissionedNode.verify(perm.removeNodesFromWhitelist(allowedNode));
+    permissionedNode.verify(perm.getNodesWhitelist(1));
 
     permissionedNode.verify(net.awaitPeerCount(1));
   }
 
+  @Test
+  public void forbiddenNodeAddedToWhitelistCanConnectToPermissionedNode() {
+    permissionedNode.verify(net.awaitPeerCount(2));
+
+    // add node to the whitelist
+    permissionedNode.verify(perm.addNodesToWhitelist(forbiddenNode));
+    permissionedNode.verify(perm.getNodesWhitelist(3));
+
+    permissionedNode.verify(admin.addPeer(((RunnableNode) forbiddenNode).enodeUrl()));
```

Comment:
Can we change `admin.addPeer` to take a `Node` instead of the node URL and then it can do the cast and extract inside the DSL.